app/screens/PartyDetailsScreen/PartyDetailsScreen.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { FC } from "react"
```

Comment:
This file includes strings for now while we wrap up the final WebFlow data types and then it should all come from there